src/components/App/App.js

```diff
@@ -4,15 +4,16 @@ import ListHeader from '../ListHeader';
 import ShoppingList from '../ShoppingList';
 import ListSelector from '../ListSelector';
 import useAppState from '../../hooks/useAppState';
+import { selectItems, selectLists } from '../../hooks/useAppState/selectors';
 
 const App = () => {
   const [state, actions] = useAppState();
 
   return (
     <>
       <ListHeader />
-      <ShoppingList items={state.items} actions={actions} />
-      <ListSelector />
+      <ShoppingList items={selectItems(state)} actions={actions} />
```

Comment:
I've created a few helper functions since the state object is a bit more complicated that retrieve specific parts out of the state